server/routes/charts.js

```diff
@@ -39,9 +40,45 @@ router.route('/charts/new')
 router.route('/charts/:chart_id')
   .get(ensureAuthenticated, async(req,res) => {
     try {
+      const data = []
+      const { dataDb, appDb } = req.app.locals
       const { chart_id } = req.params
+      const { access } =  await appDb
+        .collection(collections.users)
+        .findOne(
+          { uid: req.user },
+          { _id: 0, access: 1 }
```

Comment:
can you explain this second arg? It looks like it would be the `options` object but I don't see these fields in the [docs](https://mongodb.github.io/node-mongodb-native/3.6/api/Collection.html#findOne) (I think those are the right docs for this projects mongo version) should this be part of a projection? -- also I _thought_ you could only specify inclusions or specify exclusions in projections, but not both, maybe that's changed?
```suggestion
          projection: {
            access: 1
          }
```

Reply:
[3d5d8c](https://github.com/AMP-SCZ/dpdash/pull/91/commits/3d5d8c2367dffb4b84223b17754ce6f1a2ecb5c6)

Reply:
ah! `_id` is an exception to the rule! here's the [docs](https://www.mongodb.com/docs/manual/tutorial/project-fields-from-query-results/). So feel free to add that back in if you like 👍 

> With the exception of the _id field, you cannot combine inclusion and exclusion statements in projection documents.

Reply:
Sorry, I'll write it out next time 🙂